src/context/global-state-context.tsx

```diff
@@ -19,8 +19,8 @@ export const useFragment = (id?: string) => {
 		// this happens when rendering, which is fine because it's used
 		// in AComponent to update the state of the whole fragment due to
 		// user interaction - functionality not needed for render only
-		console.info("Rendering only, won't be able to update context");
-		return [{}, (_: any) => {}];
+		console.info('Rendering only, won\'t be able to update context');
+		return [{}, () => { }];
```

Comment:
So use a single quote only so we can escape it?
Also changes the function signature.

Reply:
Fixed single quote issue by adding an option to avoid escape for single quotes